tests/phi-ao/input.dat

```diff
@@ -0,0 +1,64 @@
+#! Test of the new phi_ao code (for puream and non-puream basis functions)
+import numpy as np
+
+REF_NELECTRON = 10.0000000 # TEST
+
+mol = psi4.geometry("""
+  0 1
+  O
+  H 1 1.0
+  H 1 1.0 2 104.5
+  symmetry c1
+  no_reorient
+  no_com
+""")
+
+TEST_NELECTRON = {}
```

Comment:
We don't need this. Just get the value in each for loop and compare it. It doesn't need to persist after that.